protocol/dataunit/client_server_test.go

```diff
@@ -94,15 +129,22 @@ func TestServerContextCancelled(t *testing.T) {
 	c := &Client{Conn: clientConn}
 	s := &Server{Conn: serverConn}
 
-	go c.ExchangeDataUnit(ctx, []byte("hello"))
+	go func() {
+		_, _ = c.ExchangeDataUnit(ctx, []byte("hello"))
+	}()
 
-	wantErr := errors.New("server context canceled")
+	wantErr := errors.New("server context cancelled")
```

Comment:
🚲🏢 (huge bike shed warning) ⚠️ ⛔️:

Turns out both spellings are correct and we have chosen the shorter one in Domainr's codebase.

Reply:
lol, I clearly don't think any other country exists outside of the UK 😄 🤦🏻 

I'll revert that.